src/collectors.m4

```diff
@@ -525,6 +526,15 @@ m4_define([_ARG_POSITIONAL_DOUBLEDASH], [m4_do(
 )])
 
 
+argbash_api([ARGBASH_INDICATE_SUPPLIED], [m4_do(
+	[_ARGBASH_INDICATE_SUPPLIED($@)],
+)])
+
+
+m4_define([_ARGBASH_INDICATE_SUPPLIED], [m4_do(
+	[m4_list_append([HAVE_SUPPLIED], $@)],
```

Comment:
Related to the suggestion to change to a set syntax, that would become `m4_set_add_all ([HAVE_SUPPLIED], $@)`.